test/jdl/grammar/validator.spec.js

```diff
@@ -503,6 +503,33 @@ describe('JDLSyntaxValidatorVisitor', () => {
                 });
             });
         });
+        context('and using for withAdminUi', () => {
+            context('a valid value', () => {
+                it('should not report a syntax error', () => {
+                    expect(() =>
+                        parse(`
+            application {
+              config {
+                withAdminUi true
+              }
+            }`)
+                    ).not.to.throw();
+                });
+            });
+
+            context('an invalid value', () => {
+                it('should report a syntax error', () => {
+                    expect(() =>
+                        parse(`
+            application {
+              config {
+                withAdminUi 666
+              }
+            }`)
+                    ).to.throw(new RegExp('^A boolean literal is expected, but found: "666"'));
```

Comment:
Wouldn't the following work?

```suggestion
                    ).to.throw(/^A boolean literal is expected, but found: "666"/);
```

Reply:
Indeed we should use literal regexp, I do the modification for the whole file